services/api/src/v1/users.js

```diff
@@ -1,9 +1,9 @@
 const Router = require('@koa/router');
 const Joi = require('@hapi/joi');
-const User = require('../models/user');
 const validate = require('../middlewares/validate');
 const { authenticate, fetchUser, checkUserRole } = require('../middlewares/authenticate');
 const { NotFoundError, BadRequestError } = require('../lib/errors');
+const { models } = require('../database');
```

Comment:
apart from what I said above it feels kind of weird to be accessing the models from `database` as the two aren't directly related

Reply:
Agreed. Using your structure